packages/keyring-controller/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@metamask/utils` from `^11.2.0` to `^11.4.2` ([#6054](https://github.com/MetaMask/core/pull/6054))
```

Comment:
Nit: can we remove this under the unreleased section, I believe this was added under 22.1.0 when dealing with conflicts